app/src/pages/outside/loginPage/pageBlocks/loginBlock/loginForm/loginForm.jsx

```diff
@@ -101,6 +109,13 @@ export class LoginForm extends React.Component {
     this.state = this.calculateLoginLimitState();
   }
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
React no longer recommend using this method see [this](https://ru.reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops) link.
What do you think about `getDerivedStateFromProps` or `componentDidUpdate` lifecycle methods?